infra/lib/fhir-server-stack.ts

```diff
@@ -19,18 +19,19 @@ import { Construct } from "constructs";
 import { EnvConfig } from "./env-config";
 import { getConfig } from "./shared/config";
 import { vCPU } from "./shared/fargate";
-import { isProd, mbToBytes } from "./util";
+import { isProd, isSandbox, mbToBytes } from "./util";
 
 export function settings() {
   const config = getConfig();
-  const prod = isProd(config);
+  const isLarge = isProd(config) || isSandbox(config);
   return {
-    cpu: prod ? 2 * vCPU : 1 * vCPU,
-    memoryLimitMiB: prod ? 4096 : 2048,
-    taskCountMin: prod ? 2 : 1,
-    taskCountMax: prod ? 10 : 5,
-    minDBCap: prod ? 4 : 1,
-    maxDBCap: prod ? 32 : 8,
+    cpu: isLarge ? 2 * vCPU : 1 * vCPU,
+    memoryLimitMiB: isLarge ? 4096 : 2048,
+    taskCountMin: isLarge ? 2 : 1,
+    taskCountMax: isLarge ? 10 : 5,
+    minDBCap: isLarge ? 4 : 1,
+    maxDBCap: isLarge ? 32 : 8,
+    minSlowLogDurationInMs: 600, // https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOG-MIN-DURATION-STATEMENT
```

Comment:
should this come from this variable?

`const { minDBCap, maxDBCap, minSlowLogDurationInMs } = settings();`